docs/yul.rst

```diff
@@ -948,6 +948,11 @@ the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels a
   You need to use the ``returndatasize`` opcode to check which part of this memory area contains the return data.
   The remaining bytes will retain their values as of before the call.
 
+.. note::
```

Comment:
@cameel how about this?

Reply:
Looks reasonable. I'd just add something like "see note below" in the table to draw attention to it.

Reply:
Added.